lib/util/time.ex

```diff
@@ -154,11 +152,11 @@ defmodule Antikythera.Time do
   end
 
   defun to_epoch_milliseconds(t :: v[t]) :: integer do
-    to_gregorian_milliseconds(t) - @time_epoch_offset_milliseconds
+    to_gregorian_milliseconds(t) - Antikythera.MilliSecondsInGregorian.time_epoch_offset_milliseconds
   end
 
-  defun from_epoch_milliseconds(milliseconds :: v[integer]) :: t do
-    from_gregorian_milliseconds(milliseconds + @time_epoch_offset_milliseconds)
+  defun from_epoch_milliseconds(milliseconds :: v[Antikythera.MilliSecondsInGregorian.t]) :: t do
+    from_gregorian_milliseconds(milliseconds + Antikythera.MilliSecondsInGregorian.time_epoch_offset_milliseconds)
```

Comment:
I think that using `alias` is better.

Reply:
bc64574e3719aa06a289b2908f9c6951b2c593c8